backend/src/models/domain.ts

```diff
@@ -46,6 +46,10 @@ export class Domain extends BaseEntity {
   })
   fromRootDomain: string;
 
+  @Column({
+    nullable: true
+  })
+  subdomainSource: string;
```

Comment:
Let's add comments to each new property we add to the models (here and in service.ts).